homeassistant/components/tcp/binary_sensor.py

```diff
@@ -8,8 +8,8 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.typing import ConfigType
 
+from .common import TCP_PLATFORM_SCHEMA, TcpSensor
```

Comment:
Please also invert the inheritance order in `TcpBinarySensor` so that `BinarySensorEntity` is last.